app/views/templates/_form.html.erb

```diff
@@ -9,35 +9,22 @@
   <ul class="nav nav-tabs" data-tabs="tabs">
     <li class="active"><a id="primary_tab" href="#primary" data-toggle="tab"><%= _("Template") %></a></li>
     <%= render "#{@type_name_plural}/custom_tab_headers" unless type == 'ptable'%>
-
     <li><a id='history_tab' href="#history" data-toggle="tab"><%= _("History") %></a></li>
     <% if show_location_tab? %>
       <li><a href="#locations" data-toggle="tab"><%= _("Locations") %></a></li>
     <% end %>
     <% if show_organization_tab? %>
       <li><a href="#organizations" data-toggle="tab"><%= _("Organizations") %></a></li>
     <% end %>
+    <% if Setting[:safemode_render] %>
+      <li><a id='safemode_tab' href="#safemode" data-toggle="tab"><%= _("Safemode") %></a></li>
+    <% end %>
   </ul>
 
   <div class="tab-content">
     <div class="tab-pane active" id="primary">
       <%= text_f f, :name, :disabled => @template.locked? %>
 
-      <% if show_default? %>
-        <%= checkbox_f f, :default, :label=>_('Default'), :help_block => default_template_description %>
-      <% end -%>
-
-      <%= render "custom_tabs", :f => f if type == 'ptable' %>
```

Comment:
Why move the custom tabs and default checkbox? This is unrelated to the alerts, and spreads the ptable UI about somewhat.